<?xml version="1.0"?>
<launch>

 <!-- Leak detectors DSOR driver-->
 
 <node pkg="leak" type="leak_node" name="leaks" respawn="false" output="screen">
	<rosparam command="load" file="$(find leak)/config/config_leak.yaml"/>
</node>

</launch>
